.github/workflows/publish.yml

```diff
@@ -196,8 +188,8 @@ jobs:
     - name: Create Release notes
       working-directory: ${{ env.WORKSPACE }}
       run: |
-        ${{ github.workspace }}/auto_changelog.py build-release \
-          --repo OasisLMF \
+        ${{ github.workspace }}/update-changelog.py build-release \
+          --repo ${{ github.event.repository.name }} \
```

Comment:
by dynamically specifying `--repo` instead of `--repo OasisLMF` does it mean that we can build changelogs for different repos for which this workflow is triggered?

Reply:
This `publish.yml` file is specific to oasislmf, so in practice `github.event.repository.name` will always have the value `OasisLMF`. 

But overall it makes the script a little more robust, (in case the repo name ever needs to change.. etc)